mitm_proxy_helpers/proxy.py

```diff
@@ -177,7 +179,10 @@ def start_proxy(self, script=None, config=None):
         if not script:
             script = 'har_logging'
 
-        if script == 'har_logging':
+        if script == 'no_script':
```

Comment:
Don't we need har_logging all the time? In what situations will we need to start the proxy with no script?

Reply:
with some tests like: "Neptune Video Player - Throttle test" here: https://github.com/DADC-NMS/selenium-bdd-framework/compare/mitmproxy_throttle_test_branch#diff-67c824d875433e339d1a224369e0818a
we just need the proxy to be started with no script loaded for a passthrough connection.

one question though, in for example the selenium_bdd framework (or any framework using this library) do you think mitmproxy should be started with har_logging script or no script by default?

Reply:
Since most usage requires `har_logging` i would assume this should be the default as discussed. Tests that do not require har logging can use the `no_script` script explicitly.